src/terrain/terrain.cpp

```diff
@@ -941,7 +920,7 @@ Terrain::qb_save_debug(const std::string path) {
 void
 Terrain::qb_save(const std::string path) const {
     // Saves the tiles in this to the path specified
-    voxel_utility::to_qb(path, *this);
+    voxel_utility::to_qb(std::filesystem::path(path), *this);
```

Comment:
```suggestion
    voxel_utility::to_qb(path, *this);
```
Implicit constructor go brrrrrrrrr
Anyways why do this instead of making the `qb_save` function take a path?

Reply:
all other io uses filesystem path